lib/logstash-core.rb

```diff
@@ -0,0 +1,2 @@
+module LogStash
```

Comment:
`logstash-core` is now a self contain gem, it just happen to also share the packaging stuff in the same repo.